back/engines/commercial/multi_tenancy/app/services/multi_tenancy/side_fx_tenant_service.rb

```diff
@@ -54,10 +53,7 @@ def before_update(tenant, current_user = nil) end
     def after_update(tenant, current_user = nil)
       LogActivityJob.perform_later(tenant, 'changed', current_user, tenant.updated_at.to_i)
 
-      trigger_host_changed_effects(tenant, current_user)           if tenant.host_previously_changed?
-      trigger_lifecycle_stage_change_effects(tenant, current_user) if tenant.changed_lifecycle_stage?
```

Comment:
If I get it right, we're now triggering these on the AppConfiguration instead of on the Tenant, correct?

That makes sense to me, but did you take into account the listeners over RabbitMQ? They might still be depending on certain attributes that are no longer there, or listening on the wrong routing_key. (`tenant.action` instead of `app_configuration.action`. cl2-tenant-setup and maybe cl2-admin are listening to these events.

Except for the routing key, the payload could be different as well. The activities are sent over the event bus with a serialized version of their `item` through a separate serializer class (contained in the `External` namespace by convention, more info [here](https://www.notion.so/citizenlab/RabbitMQ-and-events-b4893cbfcfc847c997e6a86f0b7dff58#7dfbbbf29f854bc88dbd733161ed2814)), which might not be defined for AppConfiguration or not serializing some needed attributes?

Reply:
See other comment :)